pkg/config/config.go

```diff
@@ -45,7 +45,7 @@ type UCANServer struct {
 
 	Port                    uint              `mapstructure:"port" validate:"required,min=1,max=65535" flag:"port"`
 	Host                    string            `mapstructure:"host" validate:"required" flag:"host"`
-	PDPServerURL            string            `mapstructure:"pdp_server_url" validate:"required,url" flag:"pdp-server-url"`
+	PDPServerURL            string            `mapstructure:"pdp_server_url" validate:"omitempty,url" flag:"pdp-server-url"`
```

Comment:
This is an optional value, so remove the requirement it's present in the config and drop the default value as Piri can run a Ucan server w/o a PDP server.